step-repositories/step-repositories-artifact/pom.xml

```diff
@@ -39,6 +39,13 @@
             <artifactId>step-automation-packages-controller</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <!-- TODO: the default logback config doesn't work for some reason  -->
```

Comment:
The fix for that issue was merged to master, you should be able to remove this workaround now.